### ISSUE TITLE
Size are not applied to the text[BUG]

### ISSUE THREAD

**Issue by oduvanka** [vvaldersteins/ngx-svg#11]
**Describe the bug**
I add size to the text, but I don't see changes

**To Reproduce**
this is my [code](https://github.com/oduvanka/checkered-notebook/tree/master/src/app/draw):

_draw.component.html_
`<svg-text ... [size]="sizeText" [color]="colorText" ></svg-text>`

_draw.component.ts_
```
export class DrawComponent implements OnInit {
...
public sizeText: number;
...
ngOnInit() { ...  this.sizeText = 8; ... }
}
```
**Expected behavior**
I want to see a smaller text.

**Screenshots**
![fontsize](https://user-images.githubusercontent.com/45239739/77861535-37dc6f00-721e-11ea-9812-7cc5b6038e9e.PNG)

**Desktop (please complete the following information):**
 - OS: Windows 8
 - Browser Google Chrome
 - Version 80.0.3987.149

**Additional context**
Angular 8
[demo](https://oduvanka.github.io/checkered-notebook/)

**Comment by vvaldersteins** [comment]
Hi @oduvanka !

Thank you. I was able to reproduce the issue and will try to make a release either later today or tomorrow. Will inform you here.

**Comment by vvaldersteins** [comment]
Hi @oduvanka .

Bug has been fixed in release v1.0.2.

Please, let me know if the issue still persists.

**Comment by oduvanka** [comment]
Thank you, problem solved )))))))))))
